src/test/arith_uint256_tests.cpp

```diff
@@ -95,25 +96,25 @@ BOOST_AUTO_TEST_CASE( basics ) // constructors, equality, inequality
     BOOST_CHECK(ZeroL == (OneL << 256));
 
     // String Constructor and Copy Constructor
```

Comment:
This comment should be adjusted, since there is no `arith_uint256(const std::string& str)` constructor anymore.